src/ModulesServiceProvider.php

```diff
@@ -12,23 +14,25 @@ abstract class ModulesServiceProvider extends ServiceProvider
     /**
      * Booting the package.
      */
-    public function boot()
-    {
-    }
+    public function boot() {}
 
     /**
      * Register all modules.
      */
-    public function register()
-    {
-    }
+    public function register() {}
 
     /**
      * Register all modules.
      */
     protected function registerModules()
     {
-        $this->app->register(BootstrapServiceProvider::class);
+        // $this->app->register(\Nwidart\Modules\Providers\BootstrapServiceProvider::class);
```

Comment:
should this be enabled?

Reply:
This is an old version of the provider. I've commented out this section for now. If everything works well, we can go ahead and delete this line and the `BootstrapServiceProvider` file from the package.